tests/testthat/test-plot.R

```diff
@@ -0,0 +1,15 @@
+### Test for plot() occumbFit --------------------------------------------------
+test_that("plot() works for occumbFit as expected", {
+  vdiffr::expect_doppelganger(
+    title = "occumbFit",
+    fig = plot(internal_fit)
```

Comment:
Would it be better to code like occumb:::internal_fit because the file is internal?